src/node/mod.rs

```diff
@@ -636,15 +658,17 @@ mod tests {
                 node.clone(),
                 &[PlotDescription::new(
                     plot_dir.as_ref(),
-                    bytesize::ByteSize::gb(2),
+                    bytesize::ByteSize::gb(1),
```

Comment:
won't this allocate space in git workflows as well in the future when you delete the `ignore`?
We can make it like 100MB maybe, to not make jobs fail and become future-proof. 
Feel free to ignore or do it later, this was a nitpick :)

Reply:
Yeah, we can, but ci doesn't run this test anyway, so I think we can ignore it for now